.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
Can you describe the errors with python 3.5 and conda? Can we use  something alse than conda? I use Python 3.5 with the wrapper and it runs fine, so I think we should try to include in the CI, if possible in any way.